.theme/app/components/Prose/ProsePre.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <pre v-if="language !== 'mermaid'" v-bind="props"><slot/></pre>
+  <div v-else>
```

Comment:
i think this div is obsolete now, we've put it when experimenting.

remove this div, and move v-else to mermaid